.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,49 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if command is '/request_review'
+        id: check-command
+        run: |
+          if [[ $(echo "${{ github.event.comment.body }}" | tr '[:upper:]' '[:lower:]') == "/request_review"* ]]; then
+            echo "::set-output name=command::true"
+          else
+            echo "::set-output name=command::false"
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Add label and request review
+        if: steps.check-command.outputs.command == 'true'
+        run: |
+          reviewers=$(echo "${{ github.event.comment.body }}" | grep -oE "@[a-zA-Z0-9_\-]+" | grep -v "@cryostatio/reviewer(s)" | paste -sd "," -)
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
```

Comment:
This returned an error for me... I think it might be because the event is an issue and you need to checkout the branch the PR is made on

Reply:
Hmm the payload seems fine. Which errors are u seeing @mwangggg?

Reply:
```
Run reviewers=$(echo "/request_review
  reviewers=$(echo "/request_review
  " | grep -oE "@[a-zA-Z0-9_\-]+" | grep -v "@cryostatio/reviewer(s)" | paste -sd "," -)
  
  echo "Adding label 'review-requested' to the PR"
  gh pr edit 19 --add-label "review-requested"
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
Adding label 'review-requested' to the PR
failed to run git: fatal: not a git repository (or any of the parent directories): .git 
```

Reply:
maybe it's because of my fork or there's a mistake somewhere on my end

Reply:
I don't think there is a problem on your side @mwangggg. I get the same error even after applying the changes you suggested. 
```Run reviewers="@cryostatio/reviewers"
  reviewers="@cryostatio/reviewers"
  
  echo "Adding label 'review-requested' to the PR"
  gh pr edit 7 --add-label "review-requested"
  
  if [ -n "$reviewers" ]; then
    gh pr request-review [7](https://github.com/aali309/cryostat/actions/runs/6030414003/job/16362168396#step:2:7) --reviewer $reviewers
  else
    echo "Requesting review from @cryostatio/reviewer(s) team"
    gh pr request-review 7 --team cryostatio/reviewer(s)
  fi
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
Adding label 'review-requested' to the PR
failed to run git: fatal: not a git repository (or any of the parent directories): .git

Error: Process completed with exit code 1.
```

Reply:
Looks like for some reason, the `actions/checkout` step ends up checking out the source repository but there is no `.git` corresponding to the clone - at least not in the PWD of the runner. Try updating the `checkout` action version, and then check what directory it clones to or see if it can be configured to clone into the runner's PWD if it doesn't already - though I would think it does, given that normally you can just run `checkout` and then immediately start doing builds etc. in the PWD, But perhaps the sources end up in PWD and the `.git` of the repository is elsewhere... ?

Reply:
I think it could be because the `issue_comment` is not an event the the `checkout` action can normally expect (i.e. pull_request, push) and easily extract the PR info and checkout. We probably need to explicitly tell it to so like this:

https://github.com/cryostatio/cryostat/blob/8e32669daa1aa3ed57b694690013215e3803c908/.github/workflows/ci-build-image.yml#L30